cmake/CPackConfig.cmake

```diff
@@ -79,4 +79,15 @@ install(FILES "${PRECICE_PACKAGING_DIR}/lintian-override"
   RENAME ${CPACK_PACKAGE_NAME}
   )
 
+# Configure a pkg-config file for the debian package
+configure_file(
+  "${PROJECT_SOURCE_DIR}/tools/packaging/debian/precice.pc.in"
+  "${PRECICE_PACKAGING_DIR}/pkgconfig/libprecice.pc"
+  @ONLY
+  )
+install(DIRECTORY "${PRECICE_PACKAGING_DIR}/pkgconfig" 
```

Comment:
We have another issue here: our `libprecice.pc` metadata file should be installed [where pkg-config looks for it](https://superuser.com/a/47976/340502), and not where the package is installed. In principle, one could install preCICE with prefix `~/what_a_nic_dir/` and still the metadata file should go somehwere that `pkg-config` can find it.

In my system, I have such files in `/usr/lib/pkgconfig/` and `/usr/lib/x86_64-linux-gnu/pkgconfig/`, but nothing into `/usr/local/lib/x86_64-linux-gnu/pkgconfig` (where it does not look into, in my case). So, we need something different than `${PRECICE_PACKAGING_DIR}` here.

Reply:
`${PRECICE_PACKAGING_DIR}` is a directory in the binary directory for gathering processed files only.
The install destination is located in the following line and relative to the `CMAKE_INSTALL_PREFIX`